docs/content/latest/integrations/prisma.md

```diff
@@ -306,27 +308,27 @@ node index.js
 
 ## Clean up
 
-Now that you're done with this exploration, you can clean up the pieces for your next adventure.
+Now that you have completed with this exploration, you can clean up the pieces as follows:
```

Comment:
```suggestion
Now that you've completed this exploration, clean up the pieces as follows:
```

Reply:
Changed to "Now that you have completed this exploration"